src/Containers/SessionChoice/styles.ts

```diff
@@ -0,0 +1,28 @@
+import styled from 'styled-components';
+import palette from 'palette';
+
+export const Box = styled('div')`
+  padding-top: 100px;
```

Comment:
Using padding like this is a bad practice.
There are my suggestions:
```ts
// Containers/SessionChoice/styles.ts
export const CenteredContainer = styled('div')`
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  height: 100%;
  width: 100%;
`;
```

```tsx
// Containers/SessionChoice/index.tsx
const SessionChoice = () => (
  <Box>
    <CenteredContainer>
      <Row gutter={[0, 48]}>
        <Col span={12} offset={6}>
          <Row gutter={[0, 16]}>
            <Col span={24}>
              <Tags>Nickname</Tags>
              <Input />
            </Col>
            <Col span={24}>
              <Tags>
                Server Code <OptionalText>(Optional)</OptionalText>
              </Tags>
              <Input />
            </Col>
          </Row>
        </Col>
        <Col span={12} offset={6}>
          <Button>Create new session</Button>
        </Col>
      </Row>
    </CenteredContainer>
  </Box>
);
```

By doing it this way we do not use some hack tricks using the fixed margin-top value in pixels and we get the same result